webviews/dataclass-editor/vite.config.ts

```diff
@@ -0,0 +1,15 @@
+import { defineConfig } from 'vite';
+import tsconfigPaths from 'vite-tsconfig-paths';
+
+export default defineConfig(() => {
+  const config = {
+    plugins: [tsconfigPaths()],
+    build: {
+      manifest: 'build.manifest.json',
+      outDir: '../../extension/dist/webviews/dataclass-editor',
+      chunkSizeWarningLimit: 5000
+    },
+    base: './'
+  };
+  return config;
+});
```

Comment:
If you like you can also direclty return the config
```suggestion
export default defineConfig(() => ({
  plugins: [tsconfigPaths()],
  build: {
    manifest: 'build.manifest.json',
    outDir: '../../extension/dist/webviews/dataclass-editor',
    chunkSizeWarningLimit: 5000
  },
  base: './'
}));
```

Reply:
even better 👍